src/backend/distributed/citus--8.0-10--8.0-11.sql

```diff
@@ -0,0 +1,19 @@
+/* citus--8.0-10--8.0-11 */
+SET search_path = 'pg_catalog';
+
+-- Deprecated functions
+DROP FUNCTION IF EXISTS worker_foreign_file_path(text);
+DROP FUNCTION IF EXISTS worker_find_block_local_path(bigint,text[]);
+DROP FUNCTION IF EXISTS worker_fetch_query_results_file(bigint,integer,integer,text,integer);
+DROP FUNCTION IF EXISTS master_drop_distributed_table_metadata(regclass,text,text);
+REVOKE ALL ON FUNCTION create_insert_proxy_for_table(regclass,regclass) FROM PUBLIC;
+
+-- Testing functions
+REVOKE ALL ON FUNCTION check_distributed_deadlocks() FROM PUBLIC;
```

Comment:
Should we revoke all for `recover_prepared_transactions();` as well? Because, if I do `BEGIN; SELECT recover_prepared_transactions()` from any user, it blocks whole maintenance daemon due to this [lock](https://github.com/citusdata/citus/blob/master/src/backend/distributed/transaction/transaction_recovery.c#L188) blocking on the maintenance daemon.

Reply:
Makes sense